deploy/push-ecr-image.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+AWS_REGION=$(aws configure get region || echo "$AWS_REGION" || echo "$AWS_DEFAULT_REGION")
```

Comment:
Ah -- yeah, I've [added a note](https://github.com/safeinsights/setup-app/pull/1/commits/abad0c0e9a2a862b0be917dccaeb64d6de9b8a85). I've been explicitly setting my `AWS_REGION`, but tried to write this so it would work in other ways. I guess there are corner cases where it's not sufficient :sweat_smile: .